src/main/java/net/dv8tion/jda/internal/entities/automod/AutoModResponseImpl.java

```diff
@@ -66,10 +66,11 @@ public AutoModResponseImpl(Type type, Duration duration)
 
     public AutoModResponseImpl(Guild guild, DataObject json)
     {
+        DataObject metadata = json.getObject("metadata");
```

Comment:
This will throw an exception when `metadata` is missing, which is possible in the API. To handle this, you can use `optObject("metadata")` instead.

Reply:
I just pushed an Update, you reckon that would be enough? Since `getString` and `getUnsignedLong` have a defaultValue when the key doesn't exist?